### PR TITLE
docs: Small tweaks to MEAN app guide

### PR DESCRIPTION
**Pull request by LuiseV**
Clarify how to connect to MongoDB, and how to set up Quilt.

### CODE REVIEW

File: docs/DeployMEANapp.md [hunk 1]
```diff
@@ -22,13 +22,20 @@ files:
...
 ```javascript
 var mongoose = require('mongoose');
 mongoose.connect(process.env.MONGO_URI);
 ```
 
+This is already done in the `awesome-restaurant-app`, but you'll need to
+do something similar to your own MEAN app before deploying it with Quilt.
+
+For an example, see how [server.js](https://github.com/LuiseV/awesome-restaurant-app/blob/master/server.js#L10)
+in the `awesome-restaurant-app` uses the URI in [config/database.js](https://github.com/LuiseV/awesome-restaurant-app/blob/master/config/database.js) to connect to MongoDB.
```
Comment (secant) [inline docs/DeployMEANapp.md]: link to the NetSys Quilt repo

File: docs/DeployMEANapp.md [hunk 2]
```diff
@@ -94,9 +101,14 @@ If you want to change the characteristics of the VMs, go ahead and modify the
...
 ##### Deploy
-Now we're ready to deploy our MEAN stack application! In one shell, run `quilt
-daemon`, and in another, run `quilt run specs/mean/example.js` - both from the
-`quilt` directory. You should see something like this:
+Now we're ready to deploy our MEAN stack application! If you haven't already
+worked through the [GettingStarted.md](./GettingStarted.md) guide, now is a
+good time to check it out and set up your `GOPATH` and `QUILTPATH`, and
+`quilt get` the Quilt.js specs.
+
+When you're set up, run `quilt daemon` in one shell, and then run
```
Comment (tejasmanohar) [inline docs/DeployMEANapp.md]: when are we going to make a real daemon haha 🤔

File: docs/DeployMEANapp.md (hunk 1 above)
Comment (LuiseV) [inline docs/DeployMEANapp.md]: Sorry, what should link to the Quilt repo?

Comment (secant) [inline docs/DeployMEANapp.md]: sorry, didn't read the URL carefully! =)

**Comment by LuiseV** [comment]
Ready for @ejj

**Pull request merged**